### ISSUE TITLE
Command to view fields for a database table

### ISSUE THREAD

**Issue by johnbillion** [wp-cli/ideas#97]
I sometimes have the need to remind myself of the field names for a given table in WordPress, particularly for ID fields which are a mixture of `ID`, `post_id`, `term_id`, etc.

A command which is effectively a wrapper for `SHOW FIELDS FROM <table>` would be handy.

Possible syntax:

`wp db fields <table>`

The MySQL output for a `SHOW FIELDS` command looks like this:

```
mysql> show fields from wp_terms;
+------------+---------------------+------+-----+---------+----------------+
| Field      | Type                | Null | Key | Default | Extra          |
+------------+---------------------+------+-----+---------+----------------+
| term_id    | bigint(20) unsigned | NO   | PRI | NULL    | auto_increment |
| name       | varchar(200)        | NO   | MUL |         |                |
| slug       | varchar(200)        | NO   | MUL |         |                |
| term_group | bigint(10)          | NO   |     | 0       |                |
+------------+---------------------+------+-----+---------+----------------+
4 rows in set (0.01 sec)
```

**Comment by swissspidy** [comment]
See https://github.com/wp-cli/db-command/pull/100

**Comment by schlessera** [comment]
Fixed via wp-cli/db-command#100
